scripts/jenkins/cloud/ansible/roles/deploy_caasp_terraform/tasks/deploy_caasp.yml

```
@@ -260,6 +260,13 @@
     skuba_init_command_results
   failed_when: " not ('[init] configuration files written to' in skuba_init_command_results.stdout)"
 
+- name: Management Machine | Update openstack.conf if using Octavia
+  lineinfile:
+    path: "{{ caasp_openstack_path }}/openstack.conf"
+    insertafter: '\[LoadBalancer\]'
+    line: 'use-octavia=true'
+  when: hostvars[cloud_env]['ansible_distribution_release'] == '4'
```

Should this instead be?:
```suggestion
  when: octavia_enabled
```

---

I followed the way in line 146.